apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -153,6 +153,97 @@ export function InstructorAssessmentSettings({
+              </small>
+            </div>
+            <div class="mb-3">
+              <div class="mb-3 form-check">
+                <input
+                  class="form-check-input"
+                  type="checkbox"
+                  id="allow_issue_reporting"
+                  name="allow_issue_reporting"
+                  ${canEdit ? '' : 'disabled'}
+                  ${resLocals.assessment.allow_issue_reporting ? 'checked' : ''}
+                />
+                <label class="form-check-label" for="allow_issue_reporting"
+                  >Allow issue reporting</label
+                >
+                <div class="small text-muted">
+                  Whether to allow students to report issues for assessment questions.
+                </div>
+              </div>
+            </div>
+            ${resLocals.assessment.type !== 'Homework'
+              ? html` <div class="mb-3">
+                  <div class="mb-3 form-check">
+                    <input
+                      class="form-check-input"
+                      type="checkbox"
+                      id="multiple_instance"
+                      name="multiple_instance"
+                      ${canEdit ? '' : 'disabled'}
+                      ${resLocals.assessment.multiple_instance ? 'checked' : ''}
+                    />
+                    <label class="form-check-label" for="multiple_instance"
+                      >Multiple Instances</label
+                    >
+                    <div class="small text-muted">
+                      Whether to allow students to create additional instances of the assessment.
+                    </div>
+                  </div>
+                </div>`
+              : ''}
+            <div class="mb-3 form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="allow_personal_notes"
+                name="allow_personal_notes"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.assessment.allow_personal_notes ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="allow_personal_notes"
+                >Allow personal notes</label
+              >
+              <div class="small text-muted">
+                Whether students are allowed to upload personal notes for this assessment.
+              </div>
+            </div>
+            <div class="mb-3 form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="auto_close"
+                name="auto_close"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.assessment.auto_close ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="auto_close">Auto close</label>
+              <div class="small text-muted">
+                Whether to automatically close the assessment after a period of inactivity.
```

Comment:
Nit, let's be specific about how long this period is:

```suggestion
                Whether to automatically close the assessment after 6 hours of inactivity.
```